shared/src/main/scala/amf/core/client/platform/config/ParsingOptions.scala

```diff
@@ -39,4 +40,10 @@ case class ParsingOptions(private[amf] val _internal: InternalParsingOptions) {
   /** Defines the maximum amount of nesting depth that a parsed JSON or YAML can have
     */
   def setMaxJsonYamlDepth(value: Int): ParsingOptions = _internal.setMaxJsonYamlDepth(value)
+
+  /** Keep tokens when parsing with SYAML */
+  def withKeepTokens(): ParsingOptions = _internal.withKeepTokens
```

Comment:
`withLexicalTokens` ?? withKeepTokens is a shitty name

Reply:
maybe call it a vote

Reply:
why not just withTokens() ?